.travis.yml

```diff
@@ -18,15 +18,7 @@ before_cache:
   - rm -rf /home/travis/.cargo/registry
 
 before_install:
-- if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then rvm install --disable-binary "$BUILD_RUBY_VERSION"; fi
-- if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then rvm install "$BUILD_RUBY_VERSION"; fi
-- |
-  if [[ "$TRAVIS_OS_NAME" != "windows" ]]
-  then
-    rvm use "$BUILD_RUBY_VERSION"
-    mkdir -p target/debug/deps
-    cp $(ruby -e 'puts File.join(RbConfig::CONFIG["libdir"], RbConfig::CONFIG["LIBRUBY_ALIASES"].split(" ").first)') target/debug/deps
-  fi
+- . scripts/install_ruby
```

Comment:
I've been thinking about the structure of the directory and file name here.  I've come to the conclusion that the new directory and it's name is appropriate but the fact that it's for Travis CI isn't apparent in the naming.

Could you change the `scripts/install_ruby` to `scripts/travis_ci/install_ruby`?

Reply:
I agree 👍